packages/manager/modules/cloud-connect/src/cloud-connect-datacenter-extra.class.js

```diff
@@ -0,0 +1,25 @@
+export default class CloudConnectDatacenterExtra {
+  constructor(extra) {
```

Comment:
Prefer destructuring so we only use the properties we need (and we know what's required by simply having a look to the constructor)
```suggestion
  constructor({status ... }) {
```